src/lib.rs

```diff
@@ -13,9 +13,11 @@ pub use self::error::Error;
 
 mod parser;
 use self::parser::{
-    parse_ascii_metadump_response, parse_ascii_response, parse_ascii_stats_response, Response,
+    parse_ascii_metadump_response, parse_ascii_response, parse_ascii_stats_response,
+};
+pub use self::parser::{
```

Comment:
`Response` was added in to the publicly exposed enums here to cover the requirement for downstream use in [this PR](https://github.com/Shopify/async-memcached/pull/40).

Reply:
Removing this in favor of changing the return type in `add_multi` and `set_multi` to `Result<FxHashMap<&'a K, Result<(), Error>>, Error>`.  The only green-path `Status` that we get back is `Status::Stored`, which I feel can be easily replaced with a simple `Ok(())` as a 👍 signal in order to keep the response lean.  `Status::NotStored` is parsed as an error which can be interpreted by the caller.